server.py

```diff
@@ -209,29 +269,42 @@ def recieveCommand(self, data):
     #     self.player0.Send({"action": "nextPhase", "player":data["player"]})
     #     self.player1.Send({"action": "nextPhase", "player":data["player"]})
 
-    def placeLine(self, is_h, x, y, data, num):
-        #make sure it's their turn
-        if num==self.turn:
-            self.turn = 0 if self.turn else 1
-            self.player1.Send({"action":"yourturn", "torf":True if self.turn==1 else False})
-            self.player0.Send({"action":"yourturn", "torf":True if self.turn==0 else False})
-            #place line in game
-            if is_h:
-                self.boardh[y][x] = True
-            else:
-                self.boardv[y][x] = True
-            #send data and turn data to each player
-            self.player0.Send(data)
-            self.player1.Send(data)
-print("STARTING SERVER ON LOCALHOST")
+
+    # def placeLine(self, is_h, x, y, data, num):
+    #     #make sure it's their turn
+    #     if num==self.turn:
+    #         self.turn = 0 if self.turn else 1
+    #         self.player1.Send({"action":"yourturn", "torf":True if self.turn==1 else False})
+    #         self.player0.Send({"action":"yourturn", "torf":True if self.turn==0 else False})
+    #         #place line in game
+    #         if is_h:
+    #             self.boardh[y][x] = True
+    #         else:
+    #             self.boardv[y][x] = True
+    #         #send data and turn data to each player
+    #         self.player0.Send(data)
+    #         self.player1.Send(data)
+
 # try:
 # address=input("Host:Port (localhost:8002): ")
-address="localhost:8004"#"10.0.1.52:8002" this should be my local ip address anyway with the same port of 8002 but the server errors when started with the ip instad of localhost
-if not address:
-    host, port="localhost", 8004
-else:
-    host,port=address.split(":")
-boxesServe = BoxesServer(localaddr=(host, int(port)))
-while True:
-    boxesServe.tick()
-    sleep(0.01)
+
+
+def start_server() -> None:
+    '''
+    Starts server.
+    '''
+    address="localhost:8004"    # "10.0.1.52:8002" this should be my local ip address
+                                # anyway with the same port of 8002 but the server errors
+                                # when started with the ip instad of localhost
+    print("STARTING SERVER ON " + address)
+    if not address:
+        host, port="localhost", 8004
+    else:
+        host,port=address.split(":")
+    boxesServe = BoxesServer(localaddr=(host, int(port)))
+    while True:
+        boxesServe.tick()
+        sleep(0.01)
+
+if __name__ == '__main__':
```

Comment:
Header guard. If we import server.py into a different python script, it won't run all the top-level code that is now defined in start_server().

Reply:
👍